Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "Apache-2.0"
 resolver = "2"
 
 [features]
+default = ["net"]
```

Comment:
any reason to include net?

Reply:
Nope, just forgot I added it to help my editor.